build fail is real
```
======================================================================
FAIL: test_query_no_search_all (corehq.apps.userreports.tests.test_choice_provider.LocationChoiceProviderTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_choice_provider.py", line 110, in test_query_no_search_all
    self._test_query(self.choice_query_context('', limit=20, page=0))
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_choice_provider.py", line 101, in _test_query
    self.static_choice_provider.query(query_context))
AssertionError: Lists differ: [Choice(value=u'9dd6f9d75bc362... != [Choice(value=u'9dd6f9d75bc362...
First differing element 0:
Choice(value=u'9dd6f9d75bc36216b196f5bdf9d024f7', display=u'Massachusetts/Suffolke/Bostone')
Choice(value=u'9dd6f9d75bc36216b196f5bdf9d05964', display=u'Massachusetts')
```

--- a/corehq/apps/userreports/tests/test_choice_provider.py
+++ b/corehq/apps/userreports/tests/test_choice_provider.py
@@ -157,15 +157,16 @@ class LocationChoiceProviderTest(ChoiceProviderTestMixin, LocationHierarchyTestC
         delete_all_users()
         super(LocationChoiceProviderTest, cls).setUpClass()
         report = ReportConfiguration(domain=cls.domain)
-        choices = [
-            SearchableChoice(
+        choice_tuples = [
+            (location.name, SearchableChoice(
                 location.location_id,
                 location.get_path_display(),
                 searchable_text=[location.site_code, location.name]
-            )
+            ))
             for location in cls.locations.itervalues()
         ]
-        choices.sort(key=lambda choice: choice.display)
+        choice_tuples.sort()
+        choices = [choice for name, choice in choice_tuples]
         cls.web_user = WebUser.create(cls.domain, 'blah', 'password')
         cls.choice_provider = LocationChoiceProvider(report, None)
         cls.static_choice_provider = StaticChoiceProvider(choices)
@@ -195,8 +196,8 @@ class LocationChoiceProviderTest(ChoiceProviderTestMixin, LocationHierarchyTestC
                 searchable_text=[location.site_code, location.name]
             )
             for location in [
-                self.locations['Middlesex'],
                 self.locations['Cambridge'],
+                self.locations['Middlesex'],
                 self.locations['Somerville'],
             ]
         ]

